### ISSUE TITLE
setting temperature problem

### ISSUE THREAD

**Issue by Azurovec** [Kemmey/Kemmey-TeslaWatch-Public#382]
Hello. 

I installed a siri shortcut for setting temperature in car. I confirmed that the shortcut is present in shortcuts. I can run it from siri or from pressing the play buttom in shortcuts. Both methods claim success but there is no effect on the vehicle's climate control. Excited to make this work in south Texas. 😁

**Comment by Kemmey** [comment]
Hey @Azurovec 

Sorry for the delayed response!

I just tried this a couple different ways and it does seem to work properly on my model 3. However, I did notice issues in reflecting the configured value on my phone if I executed the shortcut with Siri on my watch. 

Can you confirm that a positive response from Siri does not change anything in the vehicle - if you check the actual screen in the vehicle?

If no - could you let me know what vehicle you have, and possibly if you have the hvac set to two separate temps or synced?

Cheers,

Kim

**Comment by Azurovec** [comment]
hi Kim. Ty for the reply. 2015 model s x2. Doesnt work on either regardless
of whether i execute on watch or phone despite success claimed by both. Did
i mention how excited i am to have this work? Lol.

Andrew

**Comment by Azurovec** [comment]
Sorry hvac is synced btw. Always have it that way. Also iphone 12 mini

**Comment by Kemmey** [comment]
Hah ha - so let's fix this, shall we! :-D 

First a sanity check just to verify - you can successfully set the temp with the official app - and with the actual watch app, either running on watch or phone?

Second step would be to run the shortcut on your phone, and then send me the logs - I'm not expecting much, as it reports success, but I'll try and come up with some additional logging and have a beta build out for you ASAP.

**Comment by Kemmey** [comment]
Oh, and one last sanity check - we are discussing setting the actual temperature value of the hvac, not turning it on or off?

Might I inquire - why is it super duper important to CHANGE the set temp value - and not just turning climate on or off? I've not changed the temp value since getting the car - just leave it on 22c and auto :-D 

This is a "please enlighten me to your usecase" type of question, so I can better optimize the app for how it is used around the world.

**Comment by Azurovec** [comment]
U might have done it. I created a different shortcut just for turning on
climate instead of setting the temperature and that worked. Testing on the
other vehicle. Standby. Curious why setbtemp doesn’t work but not super
necessary as u said

**Comment by Azurovec** [comment]
So it does work but next on my wishlist is the ability to set a short cut
for each car specifically instead of which ever is checked off and selected
in the app. Possible?

**Comment by Kemmey** [comment]
Improved multi vehicle support is high on my list - I'll get to it soon. Currently it's being topped by being able to stop open features reminder at specified locations for people parking in locked garages with vehicle unlocked/open 

Then I'll get to proper multi vehicle support :-D

**Comment by Kemmey** [comment]
Curious though - set temp still doesn't work? But your requirement was to turn on hvac? 

I'd like to keep banging on the set temp, as it definitely should work.

**Comment by Azurovec** [comment]
yep set temp claimed success. perhaps it was successful but it doesn't
automatically turn on climate control. Maybe it just changed the
temperature setting for the air conditioner without actually turning on hvac

**Comment by Kemmey** [comment]
Set temp is not supposed to turn on hvac - the turn on or off climate actions will do that :-D the set temp only sets the climate temp - completely disconnected from the hvac being on or off - I guess that's the source of confusion identified then! :-D

**Comment by Azurovec** [comment]
thank u for helping me get to the bottom of it. I suppose I was primed to
think that way because in the Tesla app whenever you change the temperature
it automatically turns on the HVAC system. Just a thought with future
generations of the app.

thanks again,
andrew
